src/App/DynamicProperty.cpp

```diff
@@ -256,9 +256,15 @@ bool DynamicProperty::addProperty(Property* prop)
         return false;
     }
     auto& index = props.get<0>();
+#if BOOST_VERSION < 107500
     if (index.count(prop->getName())) {
         return false;
     }
+#else
+    if (index.contains(prop->getName())) {
+        return false;
+    }
+#endif

```

Comment:
While change to contains is really beneficial, I'm not sure if it is worth adding ifdef here. Ifdef is harder to read and process than simple count. Thoughts?

Reply:
Ok, it would be easier to migrate later indeed.

Reply:
I don't mind